utilities/virt.py

```
@@ -2004,19 +2004,25 @@ def vm_instance_from_template(
+    Runs drain/cordon synchronously and waits for KubeVirt conditions to stabilize
+    before yielding.
+    """
+    hco_namespace = get_hco_namespace(admin_client=admin_client)
     try:
         LOGGER.info(f"{node_mgmt.capitalize()} the node {node.name}")
         extra_opts = "--delete-emptydir-data --ignore-daemonsets=true --force" if node_mgmt == "drain" else ""
-        run(
-            f"nohup oc adm {node_mgmt} {node.name} {extra_opts} &",
-            shell=True,
-        )
+        run(f"oc adm {node_mgmt} {node.name} {extra_opts}", shell=True)
```

Ruff (0.14.13)

`subprocess` call with `shell=True` identified, security issue

(S602)